Author: W. Martin Borgert <debacle@debian.org>
Bug-Debian: 683188
Index: git/subversion/bindings/swig/python/svn/core.py
===================================================================
--- git.orig/subversion/bindings/swig/python/svn/core.py	2013-11-03 20:02:03.000000000 +0100
+++ git/subversion/bindings/swig/python/svn/core.py	2013-11-03 20:02:48.000000000 +0100
@@ -209,7 +209,7 @@
       # read the rest of the stream
       chunks = [ ]
       while True:
-        data = svn_stream_read(self._stream, SVN_STREAM_CHUNK_SIZE)
+        data = svn_stream_read(self._stream, int(SVN_STREAM_CHUNK_SIZE))
         if not data:
           break
         chunks.append(data)
