Author: W. Martin Borgert <debacle@debian.org>
Bug-Debian: 683188
--- subversion-1.7.9.orig/subversion/bindings/swig/python/svn/core.py
+++ subversion-1.7.9/subversion/bindings/swig/python/svn/core.py
@@ -171,7 +171,7 @@
       # read the rest of the stream
       chunks = [ ]
       while True:
-        data = svn_stream_read(self._stream, SVN_STREAM_CHUNK_SIZE)
+        data = svn_stream_read(self._stream, int(SVN_STREAM_CHUNK_SIZE))
         if not data:
           break
         chunks.append(data)
