Index: b/build/buildcheck.sh
===================================================================
--- a/build/buildcheck.sh
+++ b/build/buildcheck.sh
@@ -97,7 +97,7 @@
   esac
 fi
 
-libtool=${LIBTOOL:-`./build/PrintPath glibtool libtool libtool15`}
+libtool=${LIBTOOL:-`./build/PrintPath libtoolize glibtool libtool libtool15`}
 # Extract the libtool version number: everything from the first number in
 # the version text until a hyphen or space.
 lt_pversion=`$libtool --version 2>/dev/null |
